python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -76,8 +78,10 @@ def __init__(self,
                  process_runner=subprocess,
                  update_interval_s=AUTOSCALER_UPDATE_INTERVAL_S,
                  prefix_cluster_info=False,
-                 event_summarizer=None):
+                 event_summarizer=None,
+                 log_dir="/tmp"):
```

Comment:
We already have log_dir passed in to monitor, why not pass it to autoscaler and make it without a default value?

Reply:
Currently, all the logs will go to /tmp/node_id.out which is not consistent..